070-Integrations/100-oauth.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: OAuth 2.0
```

Comment:
No, the standard is OAuth 2.0, even if we abbreviate it informally, everywhere we should use OAuth 2.0

Reply:
I don't think we should do it. OAuth2 is very different to the first protocol and it's always referred to as OAuth2 or OAuth 2.0. Also if there's a 3.0 it will be very different (it hasn't been created in the last 12 years that OAuth2 has existed).

Reply:
OAuth2 works!